src/main/java/com/google/opensesame/servlets/MentorObject.java

```diff
@@ -1,19 +1,43 @@
 package com.google.opensesame.servlets;
 
-public class MentorObject {
-  private String name;
-  private String description;
+import java.util.ArrayList;
 
-  public MentorObject(String nameInput, String descriptionInput) {
-    name = nameInput;
-    description = descriptionInput;
+public class MentorObject extends PersonObject {
+  private ArrayList<String> projectIDs;
```

Comment:
Nit: I feel that this name could be more descriptive. Consider something like `mentoringProjectIds`.

Reply:
It's best to keep variable names as short as possible while still being descriptive. I think that `projectIDs` is fine as long as there is only one type of project. If there are multiple types of projects and mentoring projects are just one type then `mentoringProjectIds` would make sense because it would disambiguate between the different possible types of project IDs.

Reply:
Yeah that makes sense. Looking again projectIDs works just fine.

Reply:
Thanks for the feedback! It's always helpful to learn more about naming conventions.